demo/tsconfig.json

```diff
@@ -23,6 +23,7 @@
         "baseUrl": ".",
         "paths": {
             "*": [
+                "./node_modules/tns-core-modules/*",
```

Comment:
I suggest not use use paths to the core modules but write the full imports (`import { .. } from "tns-core-modules/ui/view"`) because of issues with the Angular AoT compiler.
https://github.com/angular/angular-cli/issues/5618